src/appsync.ts

```diff
@@ -708,6 +715,25 @@ export interface $util {
   authType(): string;
 }
 
+export interface log {
+  /**
+   * Logs the string representation of the provided object to the requested log stream when request-level and field-level CloudWatch logging is enabled with log level ALL on an API.
+   */
+  info(obj: Record<string, any>): void;
+  /**
+   * Logs the string representation of the provided objects to the requested log stream when request-level and field-level CloudWatch logging is enabled with log level ALL on an API. This utility will replace all variables indicated by "{}" in the first input format string with the string representation of the provided objects in order.
+   */
+  info(message: string, ...rest: Record<string, any>[]): void;
+  /**
+   * Logs the string representation of the provided object to the requested log stream when field-level CloudWatch logging is enabled with log level ERROR or log level ALL on an API.
+   */
+  error(obj: Record<string, any>): void;
```

Comment:
Is it required to be an object? Can it be a primitive like string?

Reply:
you're right. it's a string.

Reply:
ah, actually, I was wrong... it does say it's an object.

> $util.log.info(Object) : Void

Reply:
The second method signature handles the string usecase, though.

Reply:
AWS's documentation uses `Object` as `any` - it's Java based, so `Object``` just means any value type can be passed in. Not `Record<string, any>`

Reply:
Confirmed

```
$util.log.info(" 1 {} 2 {}", $v1.containsKey('key'), "hi")
```

out

```
c98f529-d1b3-4a03-a5ff-22853b3caa4e INFO - 1 true 2 hi
```

Reply:
Note, anything in the first position when more than 1 argument is given appears to fail the log... (I tried passing a boolean).